bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java

```diff
@@ -363,6 +375,14 @@ public boolean isCollapsed() {
     return map;
   }
 
+  private boolean isOperationEnabled(WebElement operation){
+    if(operation != null){
```

Comment:
Null check is redundant, `findElement()` will never return a null - the whole expression can be simplified.